examples/billboard/billboard.py

```diff
@@ -0,0 +1,187 @@
+    print("Created dataset {}".format(dataset_id))
+
+
+# Creates the view for the Billboard
+def create_billboard_view(args,isStandard):
+
+    global output_url
+
+    if isStandard == True:
+        source_id="{}.{}.{}".format(args.PROJECT_ID,args.STANDARD_BILLING_EXPORT_DATASET_NAME,args.standard_table)
+        view_id="{}.{}.{}".format(args.PROJECT_ID,args.BILLBOARD_DATASET_NAME_TO_BE_CREATED,args.bb_standard)
+    else:
+        source_id="{}.{}.{}".format(args.PROJECT_ID,args.DETAILED_BILLING_EXPORT_DATASET_NAME,args.detailed_table)
+        view_id="{}.{}.{}".format(args.PROJECT_ID,args.BILLBOARD_DATASET_NAME_TO_BE_CREATED,args.bb_detailed)
+  
+    #Check resource detailed export is present
+    try:      
+        bq_client.get_table(source_id)
+    except NotFound:
+        if isStandard == True:
+            print("Standard usage cost export not found="+source_id +" so skipping billboard view creation")
+            sys.exit()
+        else:
+            print("Detailed usage cost export not found="+source_id +" so skipping billboard detailed view creation")
+        return;
+
+    sql = """
+    CREATE VIEW if not exists `{}`
+    AS select *, COALESCE((SELECT SUM(x.amount) FROM UNNEST(s.credits) x),0) AS credits_sum_amount, COALESCE((SELECT SUM(x.amount) FROM UNNEST(s.credits) x),0) + cost as net_cost, EXTRACT(DATE FROM _PARTITIONTIME) AS date from `{}` s WHERE _PARTITIONTIME >'2020-08-01'
+    """.format(
+        view_id,source_id
+    )
+
+    bq_view_client = bigquery.Client(project=args.PROJECT_ID) #not sure why this need project_id
+
+    job = bq_view_client.query(sql)  # API request.
+    job.result()  # Waits for the query to finish.
+
+    if isStandard == True:
+        output_url=report_base_url+standard_view_url.format(args.PROJECT_ID, args.BILLBOARD_DATASET_NAME_TO_BE_CREATED,args.bb_standard)
+    else:
+        output_url=output_url+detailed_view_url.format(args.PROJECT_ID, args.BILLBOARD_DATASET_NAME_TO_BE_CREATED,args.bb_detailed)
+ 
+    print(
+        'Created view {}{}.{}.{}'.format(Back.GREEN,
+            job.destination.project,
+            job.destination.dataset_id,
+            job.destination.table_id
+        )
+    )
+    print(Style.RESET_ALL)
+
+def generate_datastudio_url(args):
+    print("To view dataset, please click "+Back.GREEN +"https://console.cloud.google.com/bigquery","\n")
+
+    print(Style.RESET_ALL)
+
+    print("To launch datastudio report, please click " +Back.GREEN + output_url +"\n")
+    print(Style.RESET_ALL)
+
+def remove_billboard_dataset(args):
+    standard_view_id="{}.{}.{}".format(args.PROJECT_ID,args.BILLBOARD_DATASET_NAME_TO_BE_CREATED,args.bb_standard)
+    bq_client.delete_table(standard_view_id, not_found_ok=True)
+    print("Billboard view {} deleted.".format(standard_view_id))
+    detailed_view_id="{}.{}.{}".format(args.PROJECT_ID,args.BILLBOARD_DATASET_NAME_TO_BE_CREATED,args.bb_detailed)
```

Comment:
Separate arguments with spaces

Reply:
Formatter helped fixing this automatically